Slim/Formats/Playlists/Base.pm

```diff
@@ -55,6 +63,7 @@ sub _updateMetaData {
 			'url'        => $entry,
 			'attributes' => $attributes,
 			'readTags'   => 1,
+			'playlist'   => $playlist,
 		} );
 	}
```

Comment:
Please change this to just the following one line change:

```
		$track = Slim::Schema->updateOrCreate( {
			'url'        => $entry,
			'attributes' => $attributes,
			'readTags'   => 1,
			'playlist'   => Slim::Music::Info::isPlaylist($entry),
		} );
```

Reply:
I knew there had to be a better solution :-) Thanks!